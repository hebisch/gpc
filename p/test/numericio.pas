program numericIO(output);
  const n = 500; d = 0.12345;
  var i: integer; x,s: real;
      f: file of real;
begin
  x := 1.0; s := 0; rewrite(f);
  for i := 1 to n do
    begin write(f,x); s := s+x; x := x+d
    end ;
  writeln(s:0:5);
  reset(f); s := 0;
  while not eof(f) do
    begin read(f,x); s := s+x
    end ;
  writeln(s:0:5)
end .
