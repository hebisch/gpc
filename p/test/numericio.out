15900.38750
15900.38750
